### PR TITLE
feat(dashboard): Add Explore Menu to widgets [APP-1010]

### PR DESCRIPTION
**Pull request by billyvg**
This allows you to go to discover or events for each query in a widget

Requires ~~https://github.com/getsentry/sentry/pull/11709~~ and ~~https://github.com/getsentry/sentry/pull/11669~~

### CODE REVIEW

File: src/sentry/static/sentry/app/components/charts/baseChart.jsx [hunk 1]
```diff
@@ -120,8 +120,8 @@ class BaseChart extends React.Component {
     // x-axis and tooltips.
     isGroupedByDate: PropTypes.bool,
 
-    // How is data grouped (affects formatting of axis labels and tooltips)
-    interval: PropTypes.oneOf(['hour', 'day']),
+    // Should we render hours on xaxis instead of day?
+    shouldXAxisRenderTimeOnly: PropTypes.bool,
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/charts/baseChart.jsx]: Refactor this to be more specific about what the prop is going to do.

File: src/sentry/static/sentry/app/views/organizationDashboard/discoverQuery.jsx [hunk 2]
```diff
@@ -88,15 +112,13 @@ class DiscoverQuery extends React.Component {
     this.resetQueries();
 
     // Fetch
+    this.setState({reloading: true});
     const promises = this.queryBuilders.map(builder => builder.fetchWithoutLimit());
     let results = await Promise.all(promises);
-    let previousData = null;
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationDashboard/discoverQuery.jsx]: unused

**Comment by lynnagara** [comment]
Can you use https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/components/organizations/globalSelectionHeader/utils.jsx#L50 (or some other equivalent) to extract the nulls from the urls?

Also, does the first widget's dropdown need a name as well?

File: src/sentry/static/sentry/app/views/organizationDashboard/data/queries/topDevicesAndBrowsers.jsx [hunk 3]
```diff
@@ -1,7 +1,8 @@
 /**
- * Top Errors by Device
+ * Top Errors by Device and Browsers
  */
-const topDevices = {
+const topDevicesAndBrowsers = {
+  name: 'Top Devices and Browsers',
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/organizationDashboard/data/queries/topDevicesAndBrowsers.jsx]: Do these titles need to be wrapped in `t()`?

**Review by markstory — APPROVED** [review]

**Comment by lynnagara** [comment]
You only need to strip nulls for events, not discover - it's only routes controlled by the global selection header that are affected by this behaviour

**Pull request merged**